coda-automation/src/Cache.re

```diff
@@ -0,0 +1,44 @@
+type model =
```

Comment:
Make sense to move everything under *src* to a clearer DIR (e.g. `key-service` under services/ or something similar less ambiguous)?

Reply:
Agree. Follow up okay? I'll start an issue with all of these...

Reply:
> Yep agree - are you cool with the follow-up pr for a corresponding issue approach to keep it in the history? I agree its a lot though so open to being convinced otherwise too

Cool, sounds good on the follow-up issue and PR. 

> > Right on - mono-repo++
> > Seems like we might want to chat about what's worth migrating//merging into _mina_ (vs. leaving in coda-automation to perhaps be deprecated) mainly to remove noise and help reduce confusion around the infra src.
> 
> Yep agree - are you cool with the follow-up pr for a corresponding issue approach to keep it in the history? I agree its a lot though so open to being convinced otherwise too

Ha yea 300+ files feels intense but cool sounds good on a follow-up issue and PR.